kasa/cli.py

```diff
@@ -302,15 +346,15 @@ async def join(dev: SmartDevice, ssid, password, keytype):
 @cli.command()
 @click.option("--timeout", default=3, required=False)
 @click.option(
-    "--show-unsupported",
-    envvar="KASA_SHOW_UNSUPPORTED",
+    "--verbose",
+    envvar="KASA_VERBOSE",
     required=False,
     default=False,
     is_flag=True,
-    help="Print out discovered unsupported devices",
+    help="Print out extra information like discovery info",
```

Comment:
```suggestion
    help="Be more verbose on output",
```
Let's keep this generic, as there may be some other things we want to print out later on if this is set.